adafruit_cursorcontrol/cursorcontrol_cursormanager.py

```diff
@@ -41,18 +40,23 @@ class CursorManager:
     """Simple interaction user interface interaction for Adafruit_CursorControl.
 
     :param Cursor cursor: The cursor object we are using.
+    :param ShiftRegisterKeys shift_register_keys: Optional initialized ShiftRegisterKeys object
+      to use instead of having CursorManager initialize and control it.
     """
 
     # pylint: disable=too-many-instance-attributes
 
-    def __init__(self, cursor: Cursor) -> None:
+    def __init__(
+        self, cursor: Cursor, shift_register_keys: ShiftRegisterKeys = None
```

Comment:
This should be `Optional[ShiftRegisterKeys]`

Reply:
@tekktrik thank you! updated with the latest commit.